### PR TITLE
fix: properties and mediaUri on gallery detail

### LINKED ISSUE #5171: Show properties for rmrk2
https://beta.kodadot.xyz/rmrk2/gallery/12336941-ac5e7ba2c1384a7463-FCSORC-ORC_SOCIETY_34-00000034

image

vs

https://singular.app/collectibles/kusama/ac5e7ba2c1384a7463-FCSORC/12336941-ac5e7ba2c1384a7463-FCSORC-ORC_SOCIETY_34-00000034

image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5171

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at https://deploy-preview-5174--koda-nuxt.netlify.app/rmrk2/gallery/12336941-ac5e7ba2c1384a7463-FCSORC-ORC_SOCIETY_34-00000034
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

File: components/gallery/GalleryItemDescription.vue [hunk 1]
```diff
@@ -107,12 +104,28 @@ const { nft, nftMimeType, nftMetadata, nftImage, nftAnimation } =
...
 const activeTab = ref('0')
 
+const properties = computed(() => {
+  // we have different format between rmrk2 and the other chains
+  if (urlPrefix.value === 'rmrk2') {
+    return Object.entries(nftMetadata.value?.properties || {}).map(
+      ([key, value]) => {
+        return {
+          trait_type: key,
+          value: (value as unknown as { value: string; type: string }).value,
```
Comment (daiagi) [inline components/gallery/GalleryItemDescription.vue:114]: could you use the urlPrefix in ```useGalleryItem``` to give nftMetaData the correct typing?

File: components/gallery/useGalleryItem.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ const whichMimeType = async (data) => {
     return await getMimeType(sanitizeIpfsUrl(data.animation_url))
   } else if (data?.image) {
     return await getMimeType(sanitizeIpfsUrl(data.image))
+  } else if (data?.mediaUri) {
```
Comment (daiagi) [inline components/gallery/useGalleryItem.ts:17]: remove else if's - for easier readabilty

```suggestion
const whichMimeType = async (data) => {
  if (data?.type) {
    return data?.type
  }

  if (data?.animation_url) {
    return await getMimeType(sanitizeIpfsUrl(data.animation_url))
  }
  if (data?.image) {
    return await getMimeType(sanitizeIpfsUrl(data.image))
  }
  if (data?.mediaUri) {
    return await getMimeType(sanitizeIpfsUrl(data.mediaUri))
  }

  return ''
}```

**Review by daiagi — CHANGES_REQUESTED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by vikiival** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 35.01 USD/KSM ~ 0.571 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x8f8166ccca83f3add51de674cb89be5ff73751eb901ff9fb80fdb98c88fad37a](https://kusama.subscan.io/extrinsic/0x8f8166ccca83f3add51de674cb89be5ff73751eb901ff9fb80fdb98c88fad37a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**